arangod/RocksDBEngine/RocksDBEngine.cpp

```diff
@@ -2509,28 +2513,23 @@ Result RocksDBEngine::flushWal(bool waitForSync, bool flushColumnFamilies) {
 }
 
 void RocksDBEngine::waitForEstimatorSync(
-    std::chrono::milliseconds maxWaitTime) {
-  auto start = std::chrono::high_resolution_clock::now();
-  auto beginSeq = _db->GetLatestSequenceNumber();
-
-  while (std::chrono::high_resolution_clock::now() - start < maxWaitTime) {
-    if (_settingsManager->earliestSeqNeeded() >= beginSeq) {
-      // all synced up!
-      break;
-    }
-    std::this_thread::sleep_for(std::chrono::milliseconds(50));
-  }
+    std::chrono::milliseconds /*maxWaitTime*/) {
```

Comment:
timeout argument with this change becomes completely unused in all implementations. Maybe we should just remove this argument in general?

Reply:
Yes, that is true. We can remove the time argument as well, either in this PR or a follow-up PR.

Reply:
Pushed a change into this PR that simplifies the API.